src/Application/Commands/UpdateCharities.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MatchBot\Application\Commands;
+
+use Doctrine\Common\Cache\CacheProvider;
+use Doctrine\ORM\EntityManager;
+use Doctrine\ORM\EntityManagerInterface;
+use GuzzleHttp\Exception\TransferException;
+use MatchBot\Client\NotFoundException;
+use MatchBot\Domain\Campaign;
+use MatchBot\Domain\CampaignRepository;
+use MatchBot\Domain\CharityRepository;
+use MatchBot\Domain\DomainException\DomainCurrencyMustNotChangeException;
```

Comment:
Think there's some extraneous imports here – at least this one

Reply:
Thanks, fixed in 7e92778e .